argocd-cli.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: argocd-cli
+  version: 2.6.2
+  epoch: 0
+  description: "Declarative continuous deployment for Kubernetes."
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      license: Apache License 2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - jq
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/argoproj/argo-cd
+      branch: v${{package.version}}
+      destination: argo-cd
+  #TODO(kamesh) - simplify this using go/build pipeline
+  # but it requires few OOTB variables
+  - name: build argocd cli
+    runs: |
+      set +x
+      cd argo-cd
+      DEST_DIR="${{targets.destdir}}/usr/bin"
```

Comment:
`mkdir -p "${{targets.destdir}}/usr/bin"` ? you're not using this variable anywhere else.

Reply:
was more of copy paste from my other package, removed unwanted car